### PR TITLE
Fix Misindentation

### PR DESCRIPTION
**Pull request by ira-lb**
I'm guessing the indentation was all wrong because a separation line was missing between the rule and the ifeq statement, so the indenter mistook it to be part of the rule. added a separation line.

### CODE REVIEW

**Comment by yuval-lb** [comment]
please verify this fix build inaugurator problem.
  remove inaugurator.14 image
  build inaugurator

**Comment by ira-lb** [comment]
It fails on something else now. damnit.
```
builtin does not exist
make: *** [Makefile.build:122: build/inaugurator.thin.initrd.dir] Error 1
Makefile.build:50: recipe for target 'docker-build/inaugurator.thin.initrd.dir' failed
make[1]: *** [docker-build/inaugurator.thin.initrd.dir] Error 2
make[1]: Leaving directory '/home/ira/workspace/inaugurator'
Makefile.lb:9: recipe for target 'build' failed
make: *** [build] Error 2
```

File: Makefile.build [hunk 1]
```diff
@@ -154,10 +154,12 @@ build/build-inaugurator.dockerfile:
 	cp docker/$(@F) $@
 
 .PHONY: build_container
+
```
Comment (yuval-lb) [inline Makefile.build]: why this?
it might cause problems

Comment (ira-lb) [reply Makefile.build]: it's actually customary to stick .PHONY at the very bottom of the file. not sure why they put it where they did.

Comment (yuval-lb) [reply Makefile.build]: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

**Pull request closed without merging**